addons/mail/static/src/js/tour.js

```diff
@@ -32,8 +32,8 @@ tour.register('mail_tour', {
     },
 }, {
     trigger: ".ui-menu-item > a",
-    auto: true,
```

Comment:
What was the reason that `auto: true` didn't work?

Reply:
Yes! not working the step in manual onboarding tour so removed auto true and added the content and position for droplet